.template-lintrc.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
 module.exports = {
-  extends: 'octane'
+  extends: 'recommended',
+  rules: {
+    'require-input-label': false,
```

Comment:
All the failures here were in tests, where this isn't really needed. If you want to re-enable it, you could disable it just for test files, or fix the templates in tests. It didn't seem important enough to add more changes to this PR, though.